### ISSUE TITLE
Trext can be garbled when width changed

### ISSUE THREAD

**Issue by jeremypw** [elementary/terminal#407]

**Comment by fluks** [comment]
Do you mean the text of the command or its output that gets garbled?

I wasn't able to reproduce that, but I was able to screw bash's PS prompt with resizing the window. It didn't happen when I was in $HOME, but if I cd'd two directories deeper and then resized back and fort with mouse, from big to to very small sizes, it got garbled. I'm not sure if that is a Terminal issue.

**Comment by jeremypw** [comment]
Yes, basically its a layout problem although I seem to remember some parts of the text could get truncated.

**Comment by fluks** [comment]
gnome-terminal uses libvte and it doesn't have the problem I was able to generate.

**Comment by jeremypw** [comment]
I could reproduce the problem with Xfce Terminal.  It does require inputting a long command line that contains spacers and spans >2 lines.   When the window is resized to a small width then several command prompts appear and the command splits up into several pieces.  Restoring the original width does not properly reassemble the pieces. 

So I suspect this is not a specific elementary Terminal problem.
